frontend/src/components/SplashPage.js

```diff
@@ -35,45 +24,30 @@ const SplashPage = () => {
     const provider = new GithubAuthProvider();
     signInWithPopup(auth, provider)
       .then((result) => {
-        // This gives you a GitHub Access Token. You can use it to access the GitHub API.
```

Comment:
Same as above, HTTP POST request to backend goes here... one flow should cover both of these authentication methods